strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -738,24 +738,11 @@ def state(self, modes=None, peaks=None, **kwargs):
         # combs = it.product(*g_list)
         # covs_dict = {tuple: index for (index, tuple) in enumerate(combs)}
 
-        listmodes = list(np.concatenate((2 * np.array(modes), 2 * np.array(modes) + 1)))
-
-        covmat = self.circuit.covs
+        covmats = self.circuit.covs
```

Comment:
it is nice that we keep hbar internally since no conversion need to be made when exposing the covs and means .

Reply:
I'm not sure I understand the comment. Does something need to be changed here?

Reply:
It is just to say that I really like how this came out.